Assignment_3.md

```diff
@@ -0,0 +1,243 @@
+
+    void add(ArrayList<Employee> array_list) {
+
+        Scanner sc1 = new Scanner(System.in);
+        int n = sc1.nextInt();
+        for (int i = 0; i <= n; i++) {
+            Scanner sc = new Scanner(System.in);
+
+            System.out.println("Pkease Enter Name..!");
+            String Name = sc.nextLine();
+            System.out.println("Please Enter Email_Address..!");
+            String Email_Address = sc.nextLine();
+            System.out.println("Pkease Enter Age..!");
+            String Age = sc.nextLine();
+            System.out.println("please Enter Date_Of_Birth (dd/mm/yyyy)..!");
+            String Date_Of_Birth = sc.nextLine();
+
+            ObjectOutputStream oos = null;
+            array_list.add(new Employee(Name, Email_Address, Age, Date_Of_Birth));
+
+            try {
+                File file = new File("employees.txt");
+                fout = new FileOutputStream(file);
+                oos = new ObjectOutputStream(fout);
+                oos.writeObject(array_list);
+                oos.close();
+
+            } catch (Exception e) {
+                System.out.println(e);
+            }
+        }
+    }
+
+////////////////////////////////////////////Display/////////////////////////////
+
+    public void display(ArrayList<Employee> Employee1) {
+        ListIterator litr = null;
+
+        try {
+
+            FileInputStream fileIn = new FileInputStream("employees.txt");
+            ObjectInputStream objectIn = new ObjectInputStream(fileIn);
+            Employee1 = (ArrayList<Employee>) objectIn.readObject();
+            System.out.println("The Object has been read from the file");
+            objectIn.close();
+
+            litr = Employee1.listIterator();
+
+            System.out.println("\n Employees Data");
+            while (litr.hasNext()) {
+                System.out.println(litr.next());
+            }
+
+        } catch (Exception ex) {
+            ex.printStackTrace();
+        }
+
+    }
+////////////////////////////////////////////Search/////////////////////////////
+
+    public void search() {
+        ListIterator litr = null;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please Enter Name..!");
+        String Name = sc.nextLine();
+
+        try {
+
+            FileInputStream fileIn = new FileInputStream("employees.txt");
+            ObjectInputStream objectIn = new ObjectInputStream(fileIn);
+            ArrayList<Employee> Employee1 = (ArrayList<Employee>) objectIn.readObject();
+            System.out.println("The Object has been read from the file");
+            objectIn.close();
+
+            litr = Employee1.listIterator();
+
+            System.out.println("\n Employees Data");
+            while (litr.hasNext()) {
+                Employee emp = (Employee) litr.next();
+                if (emp.Name.equals(Name)) {
+                    System.out.println(emp);
```

Comment:
If employ found then you can break this loop

Reply:
ok sir